Lib/venv/scripts/posix/activate.csh

```diff
@@ -16,10 +16,10 @@ setenv PATH "$VIRTUAL_ENV/__VENV_BIN_NAME__:$PATH"
 setenv VIRTUAL_ENV_PROMPT "__VENV_PROMPT__"
 
 
-set _OLD_VIRTUAL_PROMPT="$prompt"
+set _OLD_VIRTUAL_PROMPT="$prompt:q"
 
 if (! "$?VIRTUAL_ENV_DISABLE_PROMPT") then
-    set prompt = "(__VENV_PROMPT__) $prompt"
+    set prompt = "(__VENV_PROMPT__) $prompt:q"
```

Comment:
The `:q` should only be added if it's not already present. Or does the shell just outright ignore duplicated `:q`?

Reply:
It does ignore duplicated :q, at least on tcsh 6.21.00 (Astron) I'm running.